template/{% if sphinx %}docs{% endif %}/explanations/decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,20 @@
+# 1. Record architecture decisions
+
+Date: 2022-02-18
+
```

Comment:
I asked @M-Chan to remove the dates from the ADRs as they're always wrong

Reply:
Couldn't find a way to generate them without making merge conflicts on each update, so thought removing them was easiest